bee-node/src/cli.rs

```diff
@@ -9,7 +9,7 @@
 // an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and limitations under the License.
 
-use crate::config::NodeConfigBuilder;
+use crate::config::BeeNodeConfigBuilder;
```

Comment:
I think we should discuss these kind of changes before doing them. I'm not sure we should do this, it breaks consistency with al the other configurations IMO.

Reply:
ehm, yea, you're probably right

Reply:
So let me change back then :disappointed: 

Reasoning for the rename:
I like to name things as concretly as possible if I can. If I know a bird is a pigeon, I don't name the type representing it `bird`, but `pigeon`. Same here, I know the node is a `Bee` node as the config is specifically designed for it.

Reply:
I usually agree but Bee is just the name of the framework. With this reasoning you should prefix absolutely everything in it with Bee.

Reply:
Also, I was not only talking about the conf, but everywhere Bee has been added, we shouldn't do that.

Reply:
alrighty, see your poinit. Removing....